src/js/modules/RoutingApp.js

```diff
@@ -25,195 +26,143 @@ import {
 } from "./core/components";
 import { DescriptionPage } from "./descriptions/components";
 import { SectionPage } from "./sections/components";
-import {
-  ContributorPage,
-  PhotographerPage,
-  IllustratorPage,
-} from "./users/components";
+import { ContributorPage, ArtistPage } from "./users/components";
 
-import { fetchAllData } from "./core/actions";
 import { sessionfy } from "./accounts/actions";
 
-class RoutingApp extends Component {
-  constructor(props) {
-    super(props);
-  }
-
-  componentDidMount() {
-    this.prepareData();
-  }
-
-  // Separate function because it is reused for the DataErrorPage
-  prepareData = () => {
-    this.props.fetchAllData();
-    const session = localStorage.getItem("session");
-    if (session) {
-      this.props.sessionfy(JSON.parse(session));
+const RoutingAppQuery = gql`

```

Comment:
Consider centralizing queries. If you add mutations, you may need to refetch data, basically reusing queries